static/js/apps/explore/app.tsx

```diff
@@ -158,14 +148,20 @@ export function App(): JSX.Element {
           >
             {formatNumber(currentTopic.meta.variableCount)}
           </span>{" "}
-          statistical variables. We collect our health information from sources
+          statistical variables. We collect our{" "}
+          {currentTopic.title.toLocaleLowerCase()} information from sources such
           such as:{" "}
           {currentTopic.meta.sources.map((s, i) => (
             <>
+              {currentTopic.meta.sources.length > 1 &&
+              i === currentTopic.meta.sources.length - 1
+                ? "and "
+                : ""}
               {s}
-              {i === currentTopic.meta.sources.length - 1 ? ", " : ""}
+              {i === currentTopic.meta.sources.length - 1 ? "" : ", "}
```

Comment:
did you mean to add comma for the last item?

Reply:
If I have the logic right, it should only add commas if it's NOT the last item, right?

Reply:
ah yes, sorry, I got original & changed mixed up